Tasks/AzureMysqlDeploymentV1/sql/MysqlClient.ts

```diff
@@ -125,7 +125,7 @@ export class MysqlClient implements ISqlClient {
             fileSourceArgument = " -e" + '"' + this._azureMysqlTaskParameter.getSqlInline() + '"';
```

Comment:
Please check if it is required for Inline flow as well.

Reply:
verified. not required for inline flow